### DISCUSSION TITLE
Blake Izayoi's Apartment

### DISCUSSION THREAD

**Discussion by Blake-Izayoi** [nzp-team/nzportable#892] [Map Releases]
To attempt to get used to the mapping software and general map creation pipeline, I've recreated my own living space as a small playable map. There's the mystery box and the four World at War perks available. The map has been tested both on PC as well as 3DS.
How long can you survive for? Please let me know your thoughts.
Credits: NZ:P Team, Killjoy.
### [DOWNLOAD](https://archive.org/download/izayoi-apartment/IzayoiApartment.zip)
![image](https://github.com/user-attachments/assets/4a58e1d9-b5de-4834-a4ab-cb8ec5f8e63a)
![image](https://github.com/user-attachments/assets/53ee95d0-5b0f-4546-9ceb-a95d018ec4df)
![image](https://github.com/user-attachments/assets/d0862f07-79a1-4430-b0fd-3948841c3c4c)

**Comment by Naievil** [comment]
Always wanted to do something like this. Looks like a lot of fun!

**Reply by Blake-Izayoi** [reply]
Thank you, that means a lot! :)

**Reply by diddy115** [reply]
i fucking love this map

**Reply by mmkkiinn** [reply]
That s map is very nice but add pap please

**Comment by MotoLegacy** [comment]
Tagging as Meets Install Standard!

**Comment by Jeyphr** [comment]
Check out my review here: https://youtu.be/Esly8zu2vSg
( When it comes out )

**Reply by Jeyphr** [reply]
It's out now

**Comment by Useless3** [comment]
I really love this map! It's perfect! But I have some suggestions.
- Secret PaP EE (Hidden PaP machine somewhere in the map)
- Secret bowie knife wallbuy
- Music(?)

**Comment by nonMat** [comment]
This was amazing! Compact but full of detail! I loved the view out of the building window!!!
<img width="1920" height="1080" alt="Blake Izayoi&#39;s Apartment" src="https://github.com/user-attachments/assets/2a4069bb-dd06-4473-bd90-3cff8b185fd7" />
